pkg/defaulting/cluster.go

```diff
@@ -211,3 +173,53 @@ func DatacenterForClusterSpec(spec *kubermaticv1.ClusterSpec, seed *kubermaticv1
 
 	return nil, field.Invalid(field.NewPath("spec", "cloud", "dc"), datacenterName, "invalid datacenter name")
 }
+
+func defaultClusterNetwork(spec *kubermaticv1.ClusterSpec) {
```

Comment:
A: If both cidrs are specified, cluster would become dual stack by default. 

Should we set  `.DualStack = true` in here? Or A is false?

Reply:
We don't have a `DualStack` field in the cluster API. Are you suggesting that we should add it?

Reply:
BTW right, is both cidrs are specified, the cluster becomes dual-stack implicitly.

Reply:
> We don't have a DualStack field in the cluster API. 
I thought it was there and we didn't set it. 

> Are you suggesting that we should add it? 
We are better without one more field to forget I guess.

Reply:
ack :)